apple-pie/styles/style.css

```diff
@@ -17,8 +17,75 @@ body, h1, h2, h3, h4, h5, h6, p, ol, ul {
 
 /*  Set the font */
 body {
-  font-family: "Helvetica", sans-serif;
+  font-family: "Roboto", sans-serif;
+  margin-left: 10%;
+  margin-right: 10%;
```

Comment:
These margins are not allowing you to see the header image in full screen. Usually body should have `margin: 0`